config/manifests/bases/docling-operator.clusterserviceversion.yaml

```
@@ -4,6 +4,12 @@ metadata:
   annotations:
     alm-examples: '[]'
     capabilities: Basic Install
+    categories: AI/ML,Document Processing
+    containerImage: quay.io/docling-project/docling-operator:latest
+    description: Docling Operator manages the lifecycle of Docling Serve instances,
+      providing document processing capabilities with AI/ML integration.
```

I don't think this works since this is on multiple miles, since we have the long description, this can be kept short.
```suggestion
    description: Docling Operator manages the lifecycle of Docling Serve instances.
```